app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>This is index page of users</h1>
+<ul class='user-list'>
+<% @users.each do |user|%>
+<li>
+  <%= render 'shared/user', user: user%>
+</li>
+<%end%>
+</ul>
```

Comment:
- [ ] Please ensure that the index page implements the design from the wireframe. You don't have to style it but kindly ensure that the wireframe design is implemented 👍 
### Wireframe design
![image](https://github.com/BranBayou/ror-blog-app/assets/46839548/f76c07e5-f786-4fa8-b36d-5ee6fd8d3099)

### Your implementation
![image](https://github.com/BranBayou/ror-blog-app/assets/46839548/eb3f98b5-244e-4733-ba00-8a49e05250fe)